Makefile

```diff
@@ -100,18 +97,6 @@ compress: build/katex.min.js build/katex.min.css
 	printf "Minified, gzipped css: %6d\n" "$${CSSSIZE}"; \
 	printf "Total:                 %6d\n" "$${TOTAL}"
 
-serve: $(NIS)
-	$(NODE) server.js
-
-flow: $(NIS)
-	node_modules/.bin/flow
-
-test: $(NIS)
-	node_modules/.bin/jest
-
-coverage: $(NIS)
-	node_modules/.bin/jest --coverage
-
```

Comment:
Do we want to remove these?  I know I'll probably type `make serve` a bunch before retraining.  Might it be better to alias them, e.g.
```
serve:
	npm start
```

Reply:
Good point.  I'll make them into aliases.